src/main/java/com/encraft/dz/items/ItemOreFinderTool.java

```diff
@@ -238,6 +238,11 @@ public void addInformation(ItemStack itemStack, EntityPlayer player, List list,
             list.add("SHIFT+RIGHT CLICK on ground to open inventory");
             list.add("You can only use 1 finder at a time");
             list.add("Search radius X, Z: "+ ConfigHandler.xzAreaRadius +" Y: "+ConfigHandler.yAreaRadius);
+
+            if (!ConfigHandler.aEnableEverywhere) {
+                list.add("** DOES NOT WORK IN SPACE! **");
+                list.add("Overworld, Nether, Twilight Forest only");
```

Comment:
it would be better if you use lang keys instead of hard coded words, so people can translate it

Reply:
Unfortunately I don't have very much experience with minecraft i18n, so I'm not exactly sure how to do this. Is there another project with functioning i18n that I can look at to get an idea on how to do this?

Reply:
Then better not add somethings you not understand :)

Reply:
Yeah -- if I was going to attempt to do i18n of this entire project, I'd probably make it a separate PR at a minimum.

Reply:
ok then